tests/lib/queries/lexical-query.test.js

```diff
@@ -379,43 +332,25 @@ describe('lexical-query.test.js', () => {
       resourceOptions: languageOptions,
       htmlSelector: testHtmlSelector,
       siteOptions: [],
-      langOpts: {},
-      maAdapter: testMaAdapter,
-      tbAdapter: Object.assign({}, testTbAdapter)
-    })
-    jest.spyOn(query.tbAdapter, 'getHomonym')
-    await query.getData()
-    expect(query.tbAdapter.getHomonym).not.toHaveBeenCalled()
-    expect(query.maAdapter.getHomonym).toHaveBeenCalledWith(mockSelector.languageID, mockSelector.normalizedText)
-  })
-
-  it('14 LexicalQuery - it finalizes if no definition requests are made', async () => {
```

Comment:
I think the finalize tests are still important, aren't they?

Reply:
Of course - Fixed it!